bin/bin/dep.ml

```diff
@@ -146,8 +146,8 @@ let process_md ~toc book_dir =
         let html = chapter ^ ".html" in
         sprintf {|(rule
   (targets %s)
-  (deps    %s)
-  (action  (run mdx output %%{deps} -o %%{targets})))|}
+  (deps    (:x %s) ../bin/bin/output.exe)
```

Comment:
it's a bit strange to depend on the private name here. Does it work if you depend on `%{exe:rwo-output}` instead? (or whatever the dune syntax is for this)

Reply:
You are right, it works perfectly and is way better !
However, the other parts of `dep.ml` are using the private name as well, do you want me to change that too in this PR ?

Reply:
you can add a commit to fix the other parts of `dep.ml` if you feel like it, but I don't think that could block the merge of that PR.

Reply:
You are right, it works perfectly and is way better ! (the right syntax is `bin`, `exe` seems to look in the current folder only)
However, the other parts of dep.ml are using the private name as well, do you want me to change that too in this PR ?